casa/OS/HostInfo.h

```diff
@@ -131,12 +131,22 @@ class HostInfo
     static ptrdiff_t swapFree();
     // </group>
 
+    // Introduce run-time memory limit re-specification capabilities
+    // <group>
+    static void setMemoryTotal(int memory);
```

Comment:
I think is not needed, because the original values are re-read from .aipsrc for each call (which by the way does not seem very efficient but I didn't change it)